src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.IncrementalMemberEditAnalyzer.cs

```diff
@@ -46,7 +46,7 @@ public void UpdateDocumentWithCachedDiagnostics(Document document)
 
             public async Task<ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<DiagnosticData>>> ComputeDiagnosticsAsync(
                 DocumentAnalysisExecutor executor,
-                ImmutableArray<AnalyzerWithState> analyzersWithState,
+                ImmutableArray<StateSet> analyzersWithState,
```

Comment:
StateSet got whittled down to just be a DiagnosticAnalyzer and a `bool IsHostAnalyzer`.  Whihc is exactly the same as the AnalyzerWithState type.  So usages of hte latter got replaced with the former.

Note: the name StateSet is terrible.  My followup pr to this will rename it.  But i didn't want to do that in this PR to keep the diff smaller.